default_app/renderer.js

```diff
@@ -1,45 +1,23 @@
 const {remote, shell} = require('electron')
-const {execFile} = require('child_process')
-
-const {execPath} = remote.process
-
-document.onclick = function (e) {
-  e.preventDefault()
-  if (e.target.tagName === 'A') {
-    shell.openExternal(e.target.href)
-  }
-  return false
-}
-
-document.ondragover = document.ondrop = function (e) {
-  e.preventDefault()
-  return false
-}
-
-const holder = document.getElementById('holder')
-holder.ondragover = function () {
-  this.className = 'hover'
-  return false
-}
-
-holder.ondragleave = holder.ondragend = function () {
-  this.className = ''
-  return false
-}
-
-holder.ondrop = function (e) {
-  this.className = ''
-  e.preventDefault()
-
-  const file = e.dataTransfer.files[0]
-  execFile(execPath, [file.path], {
-    detached: true, stdio: 'ignore'
-  }).unref()
-  return false
-}
-
-const version = process.versions.electron
-document.querySelector('.header-version').innerText = version
-document.querySelector('.command-example').innerText = `${execPath} path-to-your-app`
-document.querySelector('.quick-start-link').href = `https://github.com/electron/electron/blob/v${version}/docs/tutorial/quick-start.md`
-document.querySelector('.docs-link').href = `https://github.com/electron/electron/tree/v${version}/docs#readme`
+const path = require('path')
+const URL = require('url')
+const electronPath = path.relative(process.cwd(), remote.process.execPath)
+
+Array.from(document.querySelectorAll('a[href]')).forEach(link => {
```

Comment:
I got a minor preference for `[...document.querySelectorAll('a[href]')], but it's the same thing, so whatever you feel like!